src/classes/AccountDeDuper.cls

```diff
@@ -1,9 +1,16 @@
-public with sharing class AccountDeDuper {
+public with sharing class Account_De_Duper {
+  // new stuff
+  // more stuff
 
   // Constructor.
 Public AccountDeDuper () {
 
 }
+public void badStuff() {}
+public void badStuff2() {}
+public void BADSTUFF3() {}
+public void BADSTUFF__4() {}
+
 Public List<Account> SearchAccount (Account pCurrentRecord, String pAccountName, String pPostCode, String pCity, id pRecordTypeId, Boolean pShowAllRecordTypes, Integer pSearchLimit) {
```

Comment:
* Methods should use the `lowerCamelCase` naming convention. Please rename accordingly.
* This method has a very long parameter list. Consider encapsulating all related parameters into a single object, or consider passing some parameters in the constructor prior to the invocation of this method.